packages/ebayui-core-react/src/ebay-flag/__tests__/constants.ts

```diff
@@ -0,0 +1,273 @@
+/* This file is autogenerated, do not edit - it will be overwritten anyway */
+
+import { Icon } from "../types";
+
+export const icons: Icon[] = [
```

Comment:
Can we simply import the other file here rather than have another copy

Reply:
The other file is type definition. This one is execution and used for adding all the stories. They are both generated